proof/infoflow/ARM/ArchIRQMasks_IF.thy

```diff
@@ -155,6 +155,9 @@ lemma invoke_tcb_irq_masks[IRQMasks_IF_assms]:
     apply (simp add: option_update_thread_def | wp hoare_weak_lift_imp hoare_vcg_all_lift | wpc)+
   by fastforce+
 
+crunch arch_prepare_set_domain
+  for inv[IRQMasks_IF_assms,wp]: P
```

Comment:
It looks like this is proved both here and in ArchFinalCaps. I'm not sure what the imports look like and they might make it hard, but it would be nice not to introduce more duplication.

Reply:
There wasn't a simple common ancestor. I'll have another look.

Reply:
`IRQMasks_IF` imports `Access.ArchDomainSepInv` and is used after `FinalCaps`. I could change dependencies, but maybe not ideal?